azurerm/internal/services/network/tests/ip_group_resource_test.go

```diff
@@ -0,0 +1,265 @@
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "0"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func testCheckAzureRMIpGroupExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		client := acceptance.AzureProvider.Meta().(*clients.Client).Network.IPGroupsClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Not found: %q", resourceName)
+		}
+
+		ipGroupName := rs.Primary.Attributes["name"]
+		resourceGroup, hasResourceGroup := rs.Primary.Attributes["resource_group_name"]
+		if !hasResourceGroup {
+			return fmt.Errorf("Bad: no resource group found in state for IP Group: %q", ipGroupName)
+		}
+
+		resp, err := client.Get(ctx, resourceGroup, ipGroupName, "")
+		if err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: IP Group %q (resource group: %q) does not exist", ipGroupName, resourceGroup)
+			}
+
+			return fmt.Errorf("Bad: Get on IPGroupsClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMIpGroupDestroy(s *terraform.State) error {
+	client := acceptance.AzureProvider.Meta().(*clients.Client).Network.IPGroupsClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_ip_group" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		resp, err := client.Get(ctx, resourceGroup, name, "")
+
+		if err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return nil
+			}
+			return err
+		}
+
+		return fmt.Errorf("IP Group still exists:\n%#v", resp.IPGroupPropertiesFormat)
+	}
+
+	return nil
+}
+
+func testAccAzureRMIpGroup_basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_ip_group" "test" {
+  name                = "acceptanceTestIpGroup1"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+`, data.RandomInteger, data.Locations.Primary)
+}
+
+func testAccAzureRMIpGroup_requiresImport(data acceptance.TestData) string {
+	template := testAccAzureRMIpGroup_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_ip_group" "import" {
+  name                = azurerm_ip_group.test.name
+  location            = azurerm_ip_group.test.location
+  resource_group_name = azurerm_ip_group.test.resource_group_name
+}
+`, template)
+}
+
+func testAccAzureRMIpGroup_complete(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
we try to put the service/other identifying name in resource groups

```suggestion
  name     = "acctestRG-network-%d"
```

Reply:
Great, done!